app/models/institution.rb

```diff
@@ -242,6 +242,9 @@ def self.autocomplete(search_term, limit = 6)
     if fuzzy_search
       clause << "SIMILARITY(institution, :search_term) > #{Settings.institution_name_similarity_threshold}"
       clause << "SIMILARITY(city, :search_term) > #{Settings.institution_city_similarity_threshold}"
+      clause << 'zip = (:search_term)'
+      clause << 'ialias LIKE (:search_term)'
+      clause << 'ialias LIKE upper(:search_term)'
```

Comment:
Is this needed? I didn't think it was case-sensitive.

Reply:
You can use `:upper_search_term` instead of `upper(:search_term)` and we could also import it as upper case and not need to check twice

Reply:
There are cases where `ialias` data is stored in all caps like "CSU". Without the the upper, we won't get any results if a user searches for something like "csu".  For some reason, `upper_search_term` does not return any results from `ialias` matches when searching "csu".``

Reply:
changes are in.